modules/howtos/pages/n1ql-queries-with-sdk.adoc

```diff
@@ -84,13 +84,18 @@ Select this when consistency is always more important than performance.
 include::example$query.php[tag=scan]
 ----
 
+// Due to https://issues.couchbase.com/browse/MB-46876 this section won’t work with named scopes and collections,
```

Comment:
Are you doing the ATPLUS modifications as part of the collections-first tickets in general?
I hadn't noticed this in previous pulls - I'll make a note to fold in to .net, node, scala too if required.

Reply:
Yeah, seemed to go hand in hand since named scopes and collections caused the problem.